### PR TITLE
SUBMARINE-426. [SDK] Add Convolutional Click Prediction Model

### PR DESCRIPTION
**Pull request by aeioulisa**
### What is this PR for?
Add TensorFlow implementation of Convolutional Click Prediction Model

[CIKM 2015][A Convolutional Click Prediction Model](http://ir.ia.ac.cn/bitstream/173211/12337/1/A%20Convolutional%20Click%20Prediction%20Model.pdf)

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-426

### How should this be tested?
https://travis-ci.org/github/aeioulisa/submarine/builds/753005415

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-sdk/pysubmarine/submarine/ml/tensorflow/model/deepfm.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class DeepFM(BaseTFModel):
     def model_fn(self, features, labels, mode, params):
         super().model_fn(features, labels, mode, params)
 
+        print("mode: \n", mode)
```
Comment (pingsutw) [inline submarine-sdk/pysubmarine/submarine/ml/tensorflow/model/deepfm.py:44]:
```suggestion
```

File: submarine-sdk/pysubmarine/submarine/ml/tensorflow/model/deepfm.py [hunk 2]
```diff
@@ -55,4 +56,10 @@ def model_fn(self, features, labels, mode, params):
         with tf.variable_scope("DeepFM_out"):
             logit = linear_logit + fm_logit + deep_logit
 
+        print("------------------------ \n")
+        print("logit: \n", logit)
+        print("labels: \n", labels)
+        print("mode: \n", mode)
+        print("params: \n", params)
+        print("------------------------ \n")
```
Comment (pingsutw) [inline submarine-sdk/pysubmarine/submarine/ml/tensorflow/model/deepfm.py:64]:
```suggestion
```

**Review by pingsutw — COMMENTED** [review]
@aeioulisa Thank you for your contribution!

**Review by xunliu — COMMENTED** [review]
@aeioulisa Thank you for your contribution!

**Comment by pingsutw** [comment]
Will commit if no more comment.
@aeioulisa Thanks for your contribution.

**Pull request closed without merging**